src/pages/iou/MoneyRequestModal.js

```diff
@@ -130,35 +122,6 @@ const MoneyRequestModal = (props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps -- props.currentUserPersonalDetails will always exist from Onyx and we don't want this effect to run again
     }, []);
 
-    useEffect(() => {
-        // We only want to check if we just finished creating an IOU transaction
-        // We check it within this effect because we're sending the request optimistically but if an error occurs from the API, we will update the iou state with the error later
-        if (!prevCreatingIOUTransactionStatusRef.current || lodashGet(props.iou, 'creatingIOUTransaction')) {
-            return;
-        }
-
-        if (lodashGet(props.iou, 'error') === true) {
-            setCurrentStepIndex(0);
-        } else {
-            Navigation.dismissModal();
-        }
-    }, [props.iou]);
-
-    useEffect(() => {
```

Comment:
Wouldn't this one still be in use?

Reply:
Ah yeah this useEffect is something still works. I was looking to see how far back when we decided to add this because I wasn't sure it was being useful (just changes the currency to current location)

But I can add it back - I was a little overzealous on the refactoring and removing old code